Say Their Names/Source/Controller/Base/ServiceClientViewController.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ServiceClientViewController.swift
+//  Say Their Names
+//
+//  Created by Joseph A. Wardell on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class ServiceClientViewController : UIViewController, ServiceClient  {
```

Comment:
It is not really clear for me, the intention of this ServiceClientViewController. Your plan is to make every ViewController by default have access to the `ServiceClient`? Would you explain it?

Reply:
see my conversation with hybridcattt below.  I've removed it.